install_files/securedrop-config/etc/profile.d/securedrop_additions.sh

```diff
@@ -0,0 +1,17 @@
+[[ $- != *i* ]] && return
+
+which tmux >/dev/null 2>&1 || return
+
+function tmux_attach_via_proc() {
+    pid=$(pgrep --newest tmux)
+    if test -n "$pid"
+    then
+        /proc/$pid/exe attach
+    fi
+    return 1
+}
+
+if test -z "$TMUX"
+then
+    (tmux attach || tmux_attach_via_proc || tmux new-session)
+fi
```

Comment:
Not requesting additional comments in this file because the logic is cleanly presented and the corresponding commit message is top-notch.

Reply:
That was a good not-request, though. No one should have to go find my top-notch commit message when trying to understand this. :smile: